releases/data-center/syskit-point-2024-5/point-2024-5-64-release-note.md

```diff
@@ -0,0 +1,87 @@
+* **Already using Syskit Point?** Upgrade to the latest Syskit Point version.\
+ [![](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fsyskitassetsstorage.blob.core.windows.net%2Fpoint%2FARMTemplates%2FPointUpdateDeploy%2FPointUpdateTemplate.json)
+
+See detailed step-by-step instructions on how to upgrade Syskit Point in the [upgrade article](../../../set-up-point-data-center/deployment/upgrade-syskit-point.md).
+
+## New Features
+
+* **New feature added to Provisioning!**
+  * New section available when creating or editing Templates: **Content & Structure**
+  * The Content & Structure section provides three options:
+    * **Start from scratch** - the new provisioned workspace is blank. 
+    * **Use existing Microsoft Teams template** - when selected, predefined apps, channels, and tabs are created for the new provisioned workspace based on the template used. 
+    * **Copy from existing team** - selecting this will copy channels, apps, tabs, and settings from the selected existing workspace to the new one being provisioned.   
+  * [For more details, take a look at the Content & Structure article.](../../governance-and-automation/provisioning/content-and-structure.md)
+
+
+## Improvements & Bug Fixes
+
+* **Improvements made to reports.**
+  * **The Sites Overview report and Teams & Groups Overview report** now have an additional column to select from the column chooser: **Last Reviewed**.  
+     * This column provides information on the date and time when the last Access Review was performed on the workspace.
+  *  **The Site Overview, Storage Metrics, Site Storage Metrics reports** have a new column available to select from the column chooser: **Site Type**
+  * **The User Access and Group Access reports** now have additional columns: Sensitivity Label and Retention Label. 
+  * **The Site Storage Metrics report** is now available in all Syskit Point Subscription plans. 
+
+* **A new column** is available for workspaces in the Govern > Security & Compliance section: **Activity**.
+  * The Activity column shows the current state of the workspace: 
+    * If the light is **green**, the workspace is **active**.
+    * If the light is **gray**, the workspace is **inactive**.
+  * Activity is **based on usage** of SharePoint, Exchange, Teams and Yammer in **the last 90 days**.
+  * This column is available for both **Active Tasks and History**. 
+
+* **New Change Privacy action has been added** to the following reports: 
+  * Teams & Groups Overview, Teams & Groups Details, Sites Overview (only for Microsoft Teams & Microsoft Groups)
+  * The Change Privacy actions change the privacy of a workspace from Private to Public or vice versa. 
+
+* **The Manage Sensitivity action** on workspaces can now also be completed by Collaborators.
+  * Admins can decide which of the Manage Sensitivity actions are allowed for Collaborators: Apply, Remove, or both. 
```

Comment:
ova druga rečenica mislim da nije točna. 
ovo ide u onom smjeru da Collaboratori mogu raditi obje akcije ako im je Sens. labela publishana

Collaborators can run the sensitivity management actions (Apply, Remove) only if the selected sensitivity label was published for them in Microsoft Purview; if the sensitivity label is not published for a collaborator, they cannot manage it.